Mathlib/Topology/DiscreteSubset.lean

```diff
@@ -111,6 +111,17 @@ lemma mem_codiscreteWithin_accPt {S T : Set X} :
     S ∈ codiscreteWithin T ↔ ∀ x ∈ T, ¬AccPt x (𝓟 (T \ S)) := by
   simp only [mem_codiscreteWithin, disjoint_iff, AccPt, not_neBot]
 
+/-- If a set is codiscrete within `U`, then it is codiscrete within any subset of `U`. -/
+lemma Filter.codiscreteWithin.mono {T₁ T₂ : Set X} (hU : T₁ ⊆ T₂) :
+   codiscreteWithin T₁ ≤ codiscreteWithin T₂ := by
```

Comment:
```suggestion
/-- If a set is codiscrete within `U`, then it is codiscrete within any subset of `U`. -/
lemma Filter.codiscreteWithin.mono {U₁ U : Set X} (hU : U₁ ⊆ U) :
   codiscreteWithin U₁ ≤ codiscreteWithin U := by
```

or anything that aligns the docstring and hyp. names (i.e. `hU`) with the variable name